scripts/gha/install_prereqs_desktop.py

```diff
@@ -54,14 +54,16 @@ def main():
           # brew install go
           utils.run_command(['brew', 'install', 'go'])
 
-    # Install openssl on linux/mac if its not installed already
-    if args.ssl == 'openssl' and not utils.is_command_installed('openssl'):
```

Comment:
Is there some reason to remove this condition?
" and not utils.is_command_installed('openssl') "

Reply:
It seems openssl already exists on windows runner. With this condition, the cmd won't run.